cmd/xl-v1-healing-common.go

```diff
@@ -119,7 +119,13 @@ func listOnlineDisks(disks []StorageAPI, partsMetadata []xlMetaV1, errs []error)
 }
 
 // Returns one of the latest updated xlMeta files and count of total valid xlMeta(s) updated latest
-func getLatestXLMeta(partsMetadata []xlMetaV1, errs []error) (xlMetaV1, int) {
+func getLatestXLMeta(ctx context.Context, partsMetadata []xlMetaV1, errs []error) (xlMetaV1, error) {
+
+	// There should be atleast half correct entries, if not return failure
+	if reducedErr := reduceReadQuorumErrs(ctx, errs, objectOpIgnoredErrs, len(errs)/2); reducedErr != nil {
```

Comment:
nit: instead of `len(errs)/2` you can use `globalXLSetDriveCount/2` to indicate that quorum is half of the total disks in a set

Reply:
done